src/unistd.rs

```diff
@@ -701,6 +701,28 @@ pub fn execvp(filename: &CString, args: &[CString]) -> Result<Void> {
     Err(Error::Sys(Errno::last()))
 }
 
+/// Replace the current process image with a new one and replicate shell `PATH`
+/// searching behavior (see
+/// [execvpe(3)](http://man7.org/linux/man-pages/man3/exec.3.html)).
+///
+/// This is a combination of `::nix::unistd::execvp` and `::nix::unistd::execve`
+/// to pass an environment and have a search path. See these two for additional
+/// information.
+#[cfg(any(target_os = "haiku",
+          target_os = "linux",
+          target_os = "openbsd"))]
+#[inline]
```

Comment:
`#[inline]` is not necessary.  The compiler can figure that out on its own if the application is compiled with LTO.